sphinx/quickstart.py

```diff
@@ -1284,6 +1288,8 @@ def generate(d, overwrite=True, silent=False):
         d['extensions'] = '\n' + indent + extensions + ',\n'
     else:
         d['extensions'] = extensions
+    print(d['extensions'])
```

Comment:
It's not used for the other `ext_` values, so why is it necessary for `ext_todo`? (The replacement for `todo_include_todos` works regardless.)